src/qt/sendcoinsdialog.cpp

```diff
@@ -23,7 +23,6 @@
 #include <key_io.h>
 #include <node/ui_interface.h>
 #include <policy/fees.h>
-#include <txmempool.h>
```

Comment:
ed0a32ec66db47b99d7d89e8c6df94b3f405f8f0

This removal looks unrelated to this PR, doesn't it?

Reply:
Yeah, it's a manual removal of an unused include.
(This is an old PR, prior me knowing about IWYU existence.. sorry for that, I forgot to update it)

Do you want me to revert it or we just go ahead with it?

Reply:
I'll really appreciate addressing of comments before merging :)

Reply:
pushed